src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -7,7 +7,6 @@ import cromwell.engine.workflow.WorkflowManagerActor
 trait WorkflowManagerSystem {
   val systemName = "cromwell-system"
   implicit val actorSystem = ActorSystem(systemName)
+  lazy val dataAccess: DataAccess = DataAccess()
```

Comment:
Didn't want to start create a database pool until someone needs the actual variable. The primary consumer of this variable, `workflowManagerActor`, is `lazy`. So I made this `lazy` as well.